cmake/scripts/common/CompilerSettings.cmake

```diff
@@ -1,6 +1,6 @@
 # Languages and global compiler settings
-if(NOT DEFINED CMAKE_CXX_STANDARD OR CMAKE_CXX_STANDARD LESS 14)
-  set(CMAKE_CXX_STANDARD 14)
+if(NOT DEFINED CMAKE_CXX_STANDARD OR CMAKE_CXX_STANDARD LESS 17)
```

Comment:
This can be removed, as the special case for WindowsStore is no longer required.
The following lines also has to be removed.
https://github.com/xbmc/xbmc/blob/84bb46f393dde388031daef78724a0ed2d6f8c4e/cmake/scripts/windowsstore/ArchSetup.cmake#L82

Reply:
Thanks, updated.